php/src/Google/Protobuf/Timestamp.php

```diff
@@ -159,11 +159,11 @@ public function setNanos($var)
     /**
      * Converts PHP DateTime to Timestamp.
      *
-     * @param DateTime $datetime
+     * @param \DateTime $datetime
      */
     public function fromDateTime($datetime)
```

Comment:
Instead of doing the `if` below, you could just do:

```
public function fromDateTime(DateTime $dateTime)
```

?

Reply:
Done